packages/docusaurus-theme-classic/src/theme/DocSidebar/Desktop/Content/index.tsx

```diff
@@ -38,9 +39,14 @@ export default function DocSidebarDesktopContent({
   className,
 }: Props): JSX.Element {
   const showAnnouncementBar = useShowAnnouncementBar();
-
+  const ariaLabel = translate({
+    id: 'theme.DocSidebarDesktopContent.sidebarAriaLabel',
```

Comment:
```suggestion
    id: 'theme.docs.sidebar.navAriaLabel',
```

Reply:
Thanks a bunch, the `id` for the docs sidebar has been adjusted to `theme.docs.sidebar.navAriaLabel`